### DISCUSSION TITLE
Snowflake: Temporary tables are not shown in Database Navigator

### DISCUSSION THREAD

**Discussion by archgnu** [dbeaver/dbeaver#14494] [DBeaver Development]
When I execute this command, TESTTABLE1 is created and shown after refresh in Database Navigator
`CREATE TABLE TESTTABLE1 (ID INT);`

When I create temporary table like this, TEMPTABLE1 is not shown after refresh in Database Navigator,
`CREATE TEMPORARY TABLE TEMPTABLE1 (ID INT);`

But it is shown in Results set when I execute `SHOW TABLES` or `SELECT STATEMENT`

Is there any particular reason for that, or there are some technical limitations with showing it in Database Navigator, because I am running into the same problem when trying to implement that in test project, cannot find way to show temporary tables in my Object explorer???

**Comment by LonwoLonwo** [comment]
Hello @archgnu 

For Snowflake, DBeaver CE read tables info directly from the driver.
Commercial DBeaver read tables info from INFORMATION_SCHEMA.TABLES, but, strange thing, don't read temp tables also. It seems another driver restriction.

Unfortunately, we can't help with this problem at this moment.
